UnitySDK/Assets/ML-Agents/Editor/BroadcastHubDrawer.cs

```diff
@@ -6,33 +6,79 @@
 
 namespace MLAgents
 {
+    /// <summary>
+    /// PropertyDrawer for BroadcastHub. Defines how BroadcastHub are displayed in the
```

Comment:
"Defines how BroadcastHub are displayed in the ... " --> Used to display the BroadcastHub in the Inspector.